### ISSUE TITLE
Why are my training and test results all gray pictures

### ISSUE THREAD

**Issue by potaosking** [junyanz/pytorch-CycleGAN-and-pix2pix#1439]
I am using the default dataset
![image](https://user-images.githubusercontent.com/52370572/174480583-0c37bce1-cb45-4cbb-b937-b8a2dfb75418.png)
This is my option
----------------- Options ---------------
             aspect_ratio: 1.0
               batch_size: 1
          checkpoints_dir: ./checkpoints
                crop_size: 256
                 dataroot: ./datasets/summer2winter_yosemite    [default: None]
             dataset_mode: unaligned
                direction: AtoB
          display_winsize: 256
                    epoch: latest
                     eval: False
                  gpu_ids: 0
                init_gain: 0.02
                init_type: normal
                 input_nc: 3
                  isTrain: False                                [default: None]
                load_iter: 0                                    [default: 0]
                load_size: 256
         max_dataset_size: inf
                    model: cycle_gan                            [default: test]
               n_layers_D: 3
                     name: summer2winter_pretrained             [default: experiment_name]
                      ndf: 64
                     netD: basic
                     netG: resnet_9blocks
                      ngf: 64
               no_dropout: True
                  no_flip: False
                     norm: instance
                 num_test: 50
              num_threads: 4
                output_nc: 3
                    phase: test
               preprocess: resize_and_crop
              results_dir: ./results/
           serial_batches: False
                   suffix:
                use_wandb: False
                  verbose: False
----------------- End -------------------

Even the images I generated using the pretrained model are gray

**Comment by junyanz** [comment]
Could you share with us the training script and debugging information (loss plots)?
